paimon-format/src/main/java/org/apache/paimon/format/orc/writer/OrcBulkWriter.java

```diff
@@ -67,4 +89,43 @@ public void finish() throws IOException {
         flush();
         writer.close();
     }
+
+    @Override
+    public long length() throws IOException {
+        long estimateMemory = treeWriter.estimateMemory();
+
+        long dataLength = 0;
```

Comment:
Do we really need to get this from stripes? Can we just get pos from output stream?
All stripes should have be flushed? right?

Reply:
This is mainly the in-memory incompletely generated stripe size, the default value is 64MB or 256MB.

Reply:
Thanks, I get it.